config/template.config

```diff
@@ -8,7 +8,7 @@ includeConfig "${projectDir}/nextflow.config"
 
 // Inputs/parameters of the pipeline
 params {
-    algorithms = ['stats', 'collectwgsmetrics'] // 'stats', 'collectwgsmetrics', 'bamqc'
+    algorithm = ['fastqc', 'stats', 'collectwgsmetrics'] // 'fastqc', 'stats', 'collectwgsmetrics', 'bamqc'
```

Comment:
(**question|blocking**) Do we want to run FastQC as default (esp RG level)? Maybe prioritize the `SQC-FASTQ` pipeline? I think there's generally enough resources to run FastQC while running align-DNA on an F72.

Reply:
I was thinking since the input will sometimes be uBAMs, running FastQC on BAM files will allow consistent methods. 

And for input aligned BAMs (that are to be realigned with our methods), we can avoid writing FASTQ files and simply input the uBAM from the revertSAM step.